src/sass/_header.scss

```diff
@@ -126,14 +126,6 @@
       }
     }
   }
-  &__library {
```

Comment:
To na pewno nie jest używany kod?

Reply:
właśnie patrzyłem i wydaje się że nie jest, wszystko w sumie to samo dziedziczyło z headera

Reply:
Jak działa i się nie rozpada to wierzę na słowo xd